website/source/docs/configuration/storage/postgresql.html.md

```diff
@@ -82,6 +89,9 @@ LANGUAGE plpgsql;
   which to write Vault data. This table must already exist (Vault will not
   attempt to create it).
 
+- `lock_table` `(string: "vault_lock")` – Specifies the name of the table to use
+  for high availability locks. Like `table`, this table must already exist.
+
```

Comment:
Other doc items to add:

```md
* `lock_ttl` (string: "10s") - Specify the duration of the each lease renewal of the leader.  Once the leader has acquired the lock, it will fresh its lock halfway through its lease.  The first time the leader fails to renew its lease it will attempt every `poll_interval` until either successful or it looses its lease.
* `poll_interval` (string: "1s") - Specify how often clients should poll for the lock.
* `lock_schema` (string: "") - Specify the schema name to use.  If specified the `lock_schema` is used to provide the fully-qualified name of the `lock_table` and bypass looking for the table using the connecition's [`search_path`](https://www.postgresql.org/docs/current/static/ddl-schemas.html#DDL-SCHEMAS-PATH).
```